src/Helper/Navigation/PluginManager.php

```diff
@@ -47,7 +45,7 @@ class PluginManager extends HelperPluginManager
     /**
      * Default factories
      *
-     * @var string[]
+     * @inheritdoc
```

Comment:
I personally prefer `array<string,string>` than `inheritdoc` since using inheritdoc always depends on the tooling which is used around:
- Some IDEs are not able to "merge" informations
- Codestandard allows "inheritdoc" even if there is nothing to inherit (e.g. when the parent property is removed)

So this could become less accurate in upstream projects when changing to `inheritdoc`. Not sure if we should do this.